scripts/build.ps1

```diff
@@ -268,8 +268,8 @@ if ($Arch -eq "arm64ec") {
     if (!$IsWindows) {
         Write-Error "Arm64EC is only supported on Windows"
     }
-    if ($Tls -eq "quictls") {
-        Write-Error "Arm64EC does not support quictls"
+    if ($Tls -eq "quictls" -Or $Tls -eq "openssl") {
+        Write-Error "Arm64EC does not support quictls/openssl"
```

Comment:
Does the latest openssl still have this limitation?

Reply:
unsure, as I'm unclear on how to test it properly.  I included it to be safe, but if you have a suggestion for a good test to try with it, I'm happy to do so.